androguard/core/axml/__init__.py

```diff
@@ -821,14 +822,18 @@ def getAttributeName(self, index):
 
         res = self.sb[name]
         # If the result is a (null) string, we need to look it up.
-        if not res or res == ":":
+        if name <= len(self.m_resourceIDs):
```

Comment:
@erev0s shouldn't this be `<` instead of `<=`?

Reply:
nice catch!